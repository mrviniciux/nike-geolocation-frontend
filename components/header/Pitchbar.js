import styles from './Pitchbar.module.scss';

const Pitchbar = () => (
    <div className={styles['pitchbar-container']}>
        <div className={styles['left-arrow']}>
            <img src="assets/icon_seta_esquerda.svg" />
        </div>
        <div className={styles['content']}>
            <img src="/assets/logo_nike_snkrs.svg" />

            <span className={styles['description']}>
                Fique por dentro dos lan√ßamentos <b><u>SNKRS</u></b>
            </span>
        </div>
        <div className={styles['right-arrow']}>
            <img src="assets/icon_seta_direita.svg" />
        </div>
    </div>
);

export default Pitchbar;
