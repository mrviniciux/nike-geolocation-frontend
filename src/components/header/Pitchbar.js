import styles from './Pitchbar.module.scss';

const Pitchbar = () => (
  <div className={styles['pitchbar-container']}>
    <div data-testid="left-arrow-nav" className={styles['left-arrow']}>
      <img src="assets/icon_seta_esquerda.svg" />
    </div>
    <div className={styles.content}>
      <img data-testid="logo-img" src="/assets/logo_nike_snkrs.svg" />

      <span data-testid="pitch-description" className={styles.description}>
        Fique por dentro dos lan√ßamentos{' '}
        <b>
          <u>SNKRS</u>
        </b>
      </span>
    </div>
    <div data-testid="right-arrow-nav" className={styles['right-arrow']}>
      <img src="assets/icon_seta_direita.svg" />
    </div>
  </div>
);

export default Pitchbar;
