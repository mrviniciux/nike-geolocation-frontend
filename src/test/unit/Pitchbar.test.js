/**
 * @jest-environment jsdom
 */

import React from 'react';
import Pitchbar from '../../components/header/Pitchbar';
import { render } from '@testing-library/react';

describe('Pitchbar rendering tests/snapshot', () => {
  it('should match snapshot', () => {
    const result = render(<Pitchbar />);

    expect(result).toMatchSnapshot();
  });
});
