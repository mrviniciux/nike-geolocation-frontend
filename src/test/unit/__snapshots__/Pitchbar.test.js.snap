// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pitchbar rendering tests/snapshot should match snapshot 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div>
        <div>
          <img
            src="assets/icon_seta_esquerda.svg"
          />
        </div>
        <div>
          <img
            src="/assets/logo_nike_snkrs.svg"
          />
          <span>
            Fique por dentro dos lançamentos
             
            <b>
              <u>
                SNKRS
              </u>
            </b>
          </span>
        </div>
        <div>
          <img
            src="assets/icon_seta_direita.svg"
          />
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div>
      <div>
        <img
          src="assets/icon_seta_esquerda.svg"
        />
      </div>
      <div>
        <img
          src="/assets/logo_nike_snkrs.svg"
        />
        <span>
          Fique por dentro dos lançamentos
           
          <b>
            <u>
              SNKRS
            </u>
          </b>
        </span>
      </div>
      <div>
        <img
          src="assets/icon_seta_direita.svg"
        />
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
